integration-libs/opf/global-functions/core/facade/opf-global-functions.service.ts

```diff
@@ -0,0 +1,356 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ComponentRef,
+  Injectable,
+  NgZone,
+  ViewContainerRef,
+  inject,
+} from '@angular/core';
+import { WindowRef } from '@spartacus/core';
+import {
+  OpfErrorDialogOptions,
+  OpfKeyValueMap,
+  OpfPage,
+  defaultOpfErrorDialogOptions,
+} from '@spartacus/opf/base/root';
+import { OpfCtaFacade } from '@spartacus/opf/cta/root';
+import {
+  GlobalFunctionsDomain,
+  GlobalFunctionsInput,
+  OpfGlobalFunctionsFacade,
+} from '@spartacus/opf/global-functions/root';
+import {
+  OpfPaymentFacade,
+  OpfPaymentGlobalMethods,
+  OpfPaymentMerchantCallback,
+  OpfPaymentMethod,
+} from '@spartacus/opf/payment/root';
+import { LAUNCH_CALLER, LaunchDialogService } from '@spartacus/storefront';
+import { Observable, Subject, lastValueFrom } from 'rxjs';
+import { finalize, take } from 'rxjs/operators';
+
+@Injectable()
+export class OpfGlobalFunctionsService implements OpfGlobalFunctionsFacade {
+  protected winRef = inject(WindowRef);
+  protected ngZone = inject(NgZone);
+  protected opfPaymentFacade = inject(OpfPaymentFacade);
+  protected launchDialogService = inject(LaunchDialogService);
+  protected opfCtaFacade = inject(OpfCtaFacade);
+  protected loaderSpinnerCpntRef: void | Observable<
+    ComponentRef<any> | undefined
+  >;
+  protected _readyForScriptEvent: Subject<string> = new Subject();
+  readyForScriptEvent$: Observable<string> =
+    this._readyForScriptEvent.asObservable();
+
+  registerGlobalFunctions({
+    domain,
+    paymentSessionId,
+    vcr,
+    paramsMap,
+  }: GlobalFunctionsInput): void {
+    // SSR not supported
```

Comment:
[MINOR]
Let's explain for a future newcomer developer to this code - why it's not supported:
a) it's not suported YET (but we plan to add a support)?
vs
b) it's not supported by design - in SSR we don't want to register global functions, becasue they are meant only for the user interactions in the browser
I believe the latter.

Note: analogical remark applies to other place in this file